docs/integrations/sources/appsflyer.md

```diff
@@ -0,0 +1,9 @@
+# AppsFlyer
+
+TODO
+
+## Changelog
+
+| Version | Date       | Pull Request                                           | Subject                                     |
+| :------ | :--------- | :----------------------------------------------------- | :------------------------------------------ |
+| 0.1.0   | 2021-03-22 | [2544](https://github.com/airbytehq/airbyte/pull/2544) | Adding the appsflyer singer based connector |
```

Comment:
@Amruta-Ranade is there a place I should be listing these "placeholder" docs?

Reply:
Umm what's a placeholder doc?